source/_integrations/rainmachine.markdown

```diff
@@ -64,13 +65,22 @@ Pause all watering activities for a number of seconds. After the pause is comple
 | ---------------------- | -------- | ------------------------------ |
 | `seconds`              | no       | The number of seconds to pause |
 
+### `rainmachine.push_flow_meter_data`
+
+Push Flow Meter data from Home Assistant to the RainMachine device.
+
+| Service Data Attribute | Optional | Description                                                                                                       |
+| ---------------------- | -------- | ----------------------------------------------------------------------------------------------------------------- |
+| `value`                | no       | The flow meter value to send. May be any positive number.                                                         |
+| `unit_of_measurement`  | yes      | The flow meter units to send. String must be one of "clicks", "gal", "m3", or "litre" (default, if not provided). |
```

Comment:
The "default, if not provided" language is confusing to me.
```suggestion
| `unit_of_measurement`  | yes      | The flow meter units to send. String must be one of "clicks", "gal", "m3", or "litre" (default: "litre"). |
```

Reply:
Revised and alphabatized.